drivers/hmis/app/graphql/types/hmis_schema/referral_posting.rb

```diff
@@ -95,17 +100,40 @@ def status_note_updated_by
     end
 
     def referral_identifier
-      object.referral.identifier
+      referral.identifier
     end
 
     def referred_by
-      object.referral.service_coordinator
+      referral.service_coordinator
     end
 
     [:referral_date, :referral_notes, :chronic, :score, :needs_wheelchair_accessible_unit].each do |name|
       define_method(name) do
-        object.referral.send(name)
+        referral.send(name)
       end
     end
+
+    # FIXME: use data source
```

Comment:
What's this referring to?  Posting's reference the Project using database id (Project table `id` column, not `ProjectID` column), so there's no need to use the data source. Same goes for the enrollment reference.

Reply:
oh sorry, I was referring to the [graphql data source / loader](https://graphql-ruby.org/dataloader/sources.html). I'll change those comments to be more clear